url.bs

```diff
@@ -2066,10 +2068,10 @@ string <var>input</var>, optionally with a <a>base URL</a> <var>base</var>, opti
           <a>Windows drive letter</a>, then:
 
           <ol>
-           <li><p>If <var>url</var>'s <a for=url>host</a> is non-null,
+           <li><p>If <var>url</var>'s <a for=url>host</a> is not the empty string,
```

Comment:
Here again we're talking about a URL record so we should say "not an empty host". Same below.